### PR TITLE
Feature/gal 601

### PR DESCRIPTION
**Pull request by NeerjaGoyalNHS**
https://nhsd-jira.digital.nhs.uk/browse/GAL-601

## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [x] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by chenglawson1 — DISMISSED** [review]

File: src/app/views/invitation_planning/InvitationPlannin.test.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import React from "react";
```
Comment (Abdul-NHS) [inline src/app/views/invitation_planning/InvitationPlannin.test.js:1]: typo in name

Comment (NeerjaGoyalNHS) [reply src/app/views/invitation_planning/InvitationPlannin.test.js:1]: resolved... thanks Amar

File: src/app/views/invitation_planning/ErrorInvitations.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      tabIndex="-1"
+      onKeyDown={e => onKeyUp(e)}
+    >
+      <h2
+        class="nhsuk-error-summary__title nhsuk-u-margin-bottom-2"
+        id="error-target-title"
+      >
+        There is a problem
+      </h2>
+      <div class="nhsuk-error-summary__body">
```
[the remark this answers is not in the document]
Comment (Amar-NHS) [reply src/app/views/invitation_planning/ErrorInvitations.jsx:26]: looks like its been resolved by Neerja now.

File: src/app/views/invitation_planning/ErrorInvitations.jsx [hunk 3]
```diff
@@ -24,6 +24,8 @@ export default function Errorinvitations(props) {
         There is a problem
       </h2>
       <div class="nhsuk-error-summary__body">
+        <ul class="nhsuk-list nhsuk-error-summary__list" role="list">
+          <li>
```
[the remark this answers is not in the document]
Comment (Amar-NHS) [reply src/app/views/invitation_planning/ErrorInvitations.jsx:28]: resolved by Neerja.

**Review by Amar-NHS — APPROVED** [review]
approved

**Review by Abdul-NHS — APPROVED** [review]
LGTM

**Review by zainm97 — APPROVED** [review]

**Pull request merged**